math-magic/components/CalculatorChild.js

```diff
@@ -0,0 +1,19 @@
+import './css/CalculatorChild.css';
+
+const Child = () => {
+  const calculatorBtn = ['AC', '+/-', '%', '÷', 7, 8, 9, '×', 4, 5, 6, '-', 1, 2, 3, '+', 0, '.', '='];
+  const totalBtn = [];
+  calculatorBtn.forEach((e) => (
+    totalBtn.push(
+      <button type="button" className="btn" id={e}>{e}</button>,
+
+    )));
```

Comment:
![image](https://github.com/misikir21/mama/assets/93039359/81bf59d5-3df5-4879-af55-86b33fab4d12)

- [ ] It would be nice if you double-check your directory because I am not able to start your app using `npm run start` 😨 Kindly fix it as suggested by the previous reviewer 👍 to make your app work as expected 😊